src/lib/web-rpde-query.js

```diff
@@ -0,0 +1,81 @@
+import Settings from './settings.js';
+import { database_pool } from './database.js';
+
+
+
+
+class rpde_query {
```

Comment:
Class name should be capitalised at least and probably no underscore

Reply:
ok